bundles/org.openhab.ui/web/src/pages/settings/things/link/link-mixin.js

```diff
@@ -0,0 +1,31 @@
+export default {
+  methods: {
+    /**
+     * Check whether the type of the given Item does not match the type of the given channel.
+     * @param {object} channel
+     * @param {object} item
+     * @return {boolean}
+     */
+    itemTypeIsNotChannelType (channel, item) {
```

Comment:
This makes code like `if (!itemTypeIsNotChannelType())` much harder to read.

I'd prefer it if this method is called `itemTypeIsChannelType`, with the corresponding inverse of the logic of course.

Reply:
To be even more nitpicky:
`channelTypeIsItemType` so the method name matches the order of the parameters, or reverse the parameter order to match the method name.

Reply:
Makes sense, I will change that.